src/api/json/catalog.json

```diff
@@ -6760,6 +6760,15 @@
       "description": "Tycho.yaml file",
       "fileMatch": ["**/tycho.yml", "**/tycho.yaml"],
       "url": "https://deployments.allegrogroup.com/tycho/schema"
+    },
+    {
+      "name": "resticprofile",
+      "description": "Configuration profiles manager for restic backup",
+      "fileMatch": [
+        "profiles.json",
```

Comment:
These patterns are a bit generic. Can you add a folder to the pattern to make it more specific to restic?

Reply:
I had the same in my mind but unfortunately `profile.yaml` in the root directory is [the default](https://creativeprojects.github.io/resticprofile/configuration/index.html) where the tool reads from. 

Luckily you can provide a path to the configuration file. What is the general approach for such tools regarding JSON schemas in this repository? Introduce a dedicated subdirectory where the config file should be put in and document it somewhere?

Reply:
@screendriver If I am understanding your question correctly, yes if a pattern is too generic, usually a dedicated parent directory is added to the `fileMatch`. For example, something like `**/*restic*/**/profiles.{json,yaml}` could work. But usually, if all "standard" potential `fileMatch`es are too generic, the `fileMatch` is removed entirely, forcing users to reference the schema themselves with `"$schema": "https://json.schemastore.org/json/...`.

Reply:
Ok. Then I think I will close this issue and reference the schema directly in my file.